clang/test/Preprocessor/time64.c

```diff
@@ -0,0 +1,12 @@
+// RUN: %clang_cc1 -E -dM -triple=i686-pc-linux-gnu < /dev/null | FileCheck -match-full-lines -check-prefix TIME32 %s
```

Comment:
You can omit `< `

Reply:
Yeah, I was wondering if it's in the `init.c` tests for a reason.